paper_slides/make.py

```diff
@@ -42,7 +42,7 @@
 
 ### FILL TABLES
 gs.tablefill(template = 'code/tables.lyx', 
-             inputs   = 'input/regression.csv', 
+             inputs.  = ['input/regression.csv', 'input/regression_subset.csv'], 
```

Comment:
@lawrencech3n This should be 

inputs = ['input/regression.csv', 'input/regression_subset.csv']

instead of 

inputs. = ['input/regression.csv', 'input/regression_subset.csv']